ee/dtr/index.md

```diff
@@ -6,61 +6,42 @@ keywords: registry, repository, images
 
 >{% include enterprise_label_shortform.md %}
 
-Docker Trusted Registry (DTR) is the enterprise-grade image storage solution
-from Docker. You install it behind your firewall so that you can securely store
-and manage the Docker images you use in your applications.
+Docker Trusted Registry (DTR) is Mirantis's enterprise-grade image storage solution. Installed behind the firewall, either on-premises or on a virtual private cloud, DTR provides a secure environment from which users can store and manage Docker images.
 
-## Image and job management
+Specifically, DTR offers a wide range of benefits that include:
+
+* [Image and job management](image-and-job-management)
+* [Availability](availability)
+* [Efficiency](efficiency)
+* [Built-in access control](built-in-access-control)
+* [Security scanning](security-scanning)
+* [Image signing](image-signing)
 
-DTR can be installed on-premises, or on a virtual private
-cloud. And with it, you can store your Docker images securely, behind your
-firewall.
+## Image and job management
 
-You can use DTR as part of your continuous integration, and continuous
-delivery processes to build, ship, and run your applications.
+ DTR has a web-based user interface that you ca use to browse images and [audit repository events](/ee/dtr/user/audit-repository-events/). With the UI, you can see which Dockerfile lines produced an image and, if security scanning is enabled, a list of all of the software installed in that image. You can also [audit jobs with the web interface](/ee/dtr/admin/manage-jobs/audit-jobs-via-ui/).
 
-DTR has a web user interface that allows authorized users in your
-organization to browse Docker images and [review repository events](/ee/dtr/user/audit-repository-events/). It even allows you to see what Dockerfile
-lines were used to produce the image and, if security scanning is enabled, to
-see a list of all of the software installed in your images. Additionally, you can now [review and audit jobs on the web interface](/ee/dtr/admin/manage-jobs/audit-jobs-via-ui/).
+DTR can serve as a Continuous Integration and Continuous Delivery (CI/CD) component, in the building, shipping, and running of applications.
 
 ## Availability
 
-DTR is highly available through the use of multiple replicas of all containers
-and metadata such that if a machine fails, DTR continues to operate and can be repaired.
+DTR is highly available through the use of multiple replicas of all containers and metadata. As such, DTR will continue to operate in the event of machine failure, thus allowing for repair.
```

Comment:
I almost wonder if we should mention scaling here.  With load balancers and multiple replicas, the HA solution also provides enhanced scaling.  But only up to a particular number of replica sets (1, 3, 5, 7 -- can't go higher than that)